projects/bp-gallery/src/components/views/contact/ContactFormView.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const ContactFormView = () => {
+  const { t } = useTranslation();
+
+  const origin: string = window.location.origin;
+  let apiPath: string;
+  const regex = RegExp('localhost');
+  if (regex.exec(origin)) {
+    apiPath = 'http://localhost:9000/api/contact';
+  } else {
+    apiPath = 'https://harz-history/api/contact';
+  }
```

Comment:
maybe use asApiPath function from app-helpers.ts?

Reply:
I agree, especially because I think `harz-history` is missing a `.de` here. This would be fixed by using `asApiPath`.